java-lib/src/main/java/com/wavefront/data/Validation.java

```diff
@@ -38,6 +40,10 @@
 
   private final static LoadingCache<String, Counter> ERROR_COUNTERS = Caffeine.newBuilder().
       build(x -> Metrics.newCounter(new MetricName("point", "", x)));
+  private final static LoadingCache<String, Counter> TRIM_COUNTERS = Caffeine.newBuilder().
+      build(x -> Metrics.newCounter(new MetricName("point", "", x)));
```

Comment:
Second cache is not needed. Reuse the same one, rename it to "COUNTERS" if you don't like ERROR_COUNTERS.